homeassistant/components/wemo/strings.json

```
@@ -6,7 +6,7 @@
       }
     },
     "abort": {
-      "single_instance_allowed": "Only a single configuration of Wemo is possible.",
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
       "no_devices_found": "No Wemo devices found on the network."
```

Please also replace this key-value-pair and adjust config_flow.py and tests if necessary, thanks :+1: 
```suggestion
      "no_devices": "[%key:common::config_flow::abort::no_devices_found%]",
```

---

thanks @springstan, done

---

The key still has to be changed from `no_devices_found` to `no_devices`. Please adjust config_flow.py and the tests to reflect that change, thanks :)